org.virt_manager.virt-viewer.yml

```diff
@@ -0,0 +1,186 @@
+app-id: org.virt_manager.virt-viewer
```

Comment:
I'm wondering if this is the best naming choice.  IIUC, typical apps I have installed appear to have the last component use uppercase, eg `org.gnome.Boxes`, `org.gnome.Totem`, `us.zoom.Zoom`.  IOW, should this be instead named `org.virt_manager.VirtViewer` as the app id

Reply:
the question here is which dbus name does the app register?

Reply:
Seems we pass 'org.virt-manager.virt-viewer'  as the application-id to the GApplication class, but we run non-unique (ie multiple proccesses allowed) , so I don't believe it gets registered on dbus.

If our current application-id choice is not following best practice i'm happy to rename it upstream

Reply:
If that's the app-id passed to GApplication, that's the right one to use.

Reply:
I think originnaly was done following the bin filename